Sources/TuistLoader/Models+ManifestMappers/CopyFileElement+ManifestMapper.swift

```diff
@@ -26,8 +26,10 @@ extension TuistGraph.CopyFileElement {
                 if FileHandler.shared.isFolder(path) {
                     logger.warning("'\(path.pathString)' is a directory, try using: '\(path.pathString)/**' to list its files")
                 } else {
-                    // FIXME: This should be done in a linter.
-                    logger.warning("No files found at: \(path.pathString)")
+                    if !path.pathString.isGlobComponent {
+                        // FIXME: This should be done in a linter.
+                        logger.warning("No files found at: \(path.pathString)")
+                    }
```

Comment:
Why have we not moved this to the new linter?

Reply:
Code wise this just needs to be removed, the linter already does this